src/variable.jl

```diff
@@ -410,6 +410,98 @@ end
 
 SymbolicIndexingInterface.getname(x, val=_fail) = _getname(unwrap(x), val)
 
+function SymbolicIndexingInterface.symbolic_evaluate(ex::Union{Num, Arr, Symbolic, Equation, Inequality}, d::Dict; kwargs...)
+    fixpoint_sub(ex, d; kwargs...)
+end
+
+function fixpoint_sub(x, dict; operator = Nothing)
```

Comment:
document?

Reply:
Fixed in https://github.com/JuliaSymbolics/Symbolics.jl/pull/1089/commits/33ce32b6304e22a59f0df54484bb75be93e94dec